Person.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+class Person
+{
+    private string $lastname;
+    private string $firstname;
+    private string $adress;
+    private string $birthDate;
+
+    public function __construct(string $lastname, string $firstname, string $adress, string $birthDate)
```

Comment:
`public function __construct(private string $lastname, private string $firstname, private string $adress, private string $birthDate)`
no need for lines 5 to 8